public/app/features/explore/hooks/useStateSync/migrators/types.ts

```diff
@@ -4,7 +4,7 @@ export interface MigrationHandler<From extends BaseExploreURL | never, To> {
   /**
    * The parse function is used to parse the URL parameters into the state object.
    */
-  parse: (params: UrlQueryMap) => To;
+  parse: (params: UrlQueryMap) => [To, boolean];
```

Comment:
nit: I think it'd be more readable if we returned an object not an array

Reply:
agree!